python/ray/train/torch/config.py

```diff
@@ -210,3 +236,96 @@ def on_training_start(
         self, worker_group: WorkerGroup, backend_config: BackendConfig
     ):
         worker_group.execute(_set_torch_distributed_env_vars)
+
+
+class _TorchXLABackend(Backend):
+    """Not implemented yet."""
```

Comment:
Will this be supported in this PR? Should we instead only restrict everything to Neuron for now to keep it simple and concise, and extend support for this in the future?

Reply:
Added for future use-case and I was expecting some generic setup for TPU and NeuronCores as they both use TorchXLA to some extent. Removed for now.